[CUBRIDMAN-299] Modify english, korean and delete oledb.rst file (#634)
CUBRID/cubrid-manual: en/admin/config.rst

**cubrid_port_id**

>     **cubrid_port_id** is a parameter to configure the port to be used by the master process. The default value is **1523**. If the port 1523 is already being used on the server where CUBRID is installed or it is blocked by a firewall, an error message, which means the master server is not connected because the master process cannot be running properly, is displayed. If such port conflict occurs, the administrator must change the value of **cubrid_port_id** considering the server environment.

Reviewer: ```suggestion
    **cubrid_port_id** is a parameter to configure the port to be used by the master process. The default value is **1523**. If port 1523 is already being used on the server where CUBRID is installed or is blocked by a firewall, an error message is displayed indicating that the master server is not connected because the master process cannot run properly. If such a port conflict occurs, the administrator must change the value of **cubrid_port_id** considering the server environment.
```

    **JDBC_CACHE**, **JDBC_CACHE_HINT_ONLY**, and **JDBC_CACHE_LIFE_TIME** parameters are meaningless

> 	when the query cache-related parameter both of **max_query_cache_entries** and **query_cache_size_in_pages** are not set to positive value.

Reviewer 2: 뒤에 parameter 명을 명시해 두었기때문에 system parameter 대신 query cache-related parameter 라고 다시 명시할 필요는 없어보이는데, 원래 문장으로 원복하는 것은 어떨까요 ?

> 	when the query cache-related parameter both of **max_query_cache_entries** and **query_cache_size_in_pages** are not set to positive value.

Reviewer 2: ```suggestion
	when the query cache-related system parameter both of **max_query_cache_entries** and **query_cache_size_in_pages** are not set to positive value.
```
